src/settings.js

```diff
@@ -377,6 +381,16 @@ const types = {
   },
 };
 
+// Specifies what to do with a metadata field.
+const metaDataField = (v, _) => {
```

Comment:
```suggestion
const metaDataField = v => {
```

You could just omit unused trailing parameters; arity is not checked in JS.